imgui.ini

```diff
@@ -0,0 +1,27 @@
+[Window][Debug##Default]
```

Comment:
Is this file really needed to get checked into the repo? Usually dearimgui will do a standard layout and will persist to ini if you move things around. Or did you completely do the layout in the init only?

Reply:
No sorry, it's a mistake, I will remove it.